app/views/users/pbs/export/panels/index.erb

```diff
@@ -3,12 +3,12 @@
 <% longest[:game_name] = (['game'.length] + @runs.map { |run| run.game.name.length }).max || 'Game'.length %>
 <% longest[:category_name] = (['category'.length] + @runs.map { |run| run.category.name.length }).max || 'Category'.length %>
 <% longest[:time] = (['pb'.length] + @runs.map { |run| "[#{format_ms(run.duration_ms(run.default_timing))}](#{link(run)})".length }).max || 'PB'.length %>
-<% longest[:video_url] = (['video'.length] + @runs.map { |run| "[Watch](#{run.video_url})".length }).max || 'Video'.length %>
-<% if @runs.where.not(video_url: nil).count > 0 %>
+<% longest[:video_url] = (['video'.length] + @runs.map { |run| "[Watch](#{run.video&.url})".length }).max || 'Video'.length %>
+<% if @runs.joins(:video).where.not(videos: { url: nil }).count > 0 %>
```

Comment:
```ruby
@runs.joins(:video)
```
seems like it should already only return runs with videos, so I think the `where` is redundant.

Reply:
Agreed.